speech/cloud-client/beta_snippets.py

```diff
@@ -156,33 +152,46 @@ def transcribe_file_with_diarization(speech_file):
 
     config = speech.types.RecognitionConfig(
         encoding=speech.enums.RecognitionConfig.AudioEncoding.LINEAR16,
-        sample_rate_hertz=16000,
+        sample_rate_hertz=8000,
         language_code='en-US',
         enable_speaker_diarization=True,
         diarization_speaker_count=2)
 
     print('Waiting for operation to complete...')
     response = client.recognize(config, audio)
 
-    for i, result in enumerate(response.results):
-        alternative = result.alternatives[0]
-        print('-' * 20)
-        print('First alternative of result {}: {}'
-              .format(i, alternative.transcript))
-        print('Speaker Tag for the first word: {}'
-              .format(alternative.words[0].speaker_tag))
+    # The transcript within each result is separate and sequential per result.
+    # However, the words list within an alternative (for whatever reason)
```

Comment:
I understand that this comment is to explain why the `[-1]`, but I'm having trouble understanding what this is referring to. (Probably because I don't know what an "alternative" means in this context.)

I think you could probably get by with fewer words within the sample with just
```
# To get all words with speaker tags, you only have to take the words list from the last result.
```
and leave the extra explanation about alternatives for the actual cloud.google.com docs.

Reply:
I initially had a shorter comment. But Jerjou suggested to explain it more, so I added the longer comment. I am okay either way (just want to wrap up this PR).